bbb-webhooks/.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules/
 config_local.coffee
 config_local.js
```

Comment:
Would you please remove these two, since they are no longer used?

Reply:
Almost did this originally, but wasn't sure if there would be potential lingering configurations floating around out there that we would still like to keep out of the repository.